uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerHost.java

```diff
@@ -282,4 +282,16 @@ public String getAccountUpn() {
             UiAutomatorUtils.handleButtonClick("android:id/button1");
         }
     }
+
+    @Override
+    public void setFlights(@Nullable final String flightsJson) {
+        Logger.i(TAG, "Set Flights..");
+        launch();
+        // Make the set flights UI visible on screen
+        UiAutomatorUtils.obtainChildInScrollable("Set Flights");
+        // input flights string in flights input box
+        UiAutomatorUtils.handleInput("com.microsoft.identity.testuserapp:id/editTextFlights", flightsJson);
```

Comment:
Should we do something special for null, or will it just set the box to ""?

Reply:
it sets it to ""